src/common/primitive_attr.cpp

```diff
@@ -374,6 +374,25 @@ bool post_ops_t::check_sum_consistency(const data_type_t dst_dt,
             && check_sum_consistent_quantization(dst_dt, is_int8);
 }
 
+status_t post_ops_t::entry_t::validate_binary_with_dst_consistency(
```

Comment:
(random spot) Resampling also has binary post-op support and does not seem to be covered by the check. Is there a particular reason?